### ISSUE TITLE
Reword notes in "Mixed Content"

### ISSUE THREAD

**Issue by fred-wang** [WICG/private-network-access#57]
From https://wicg.github.io/private-network-access/#mixed-content we have:

> Note: Doing something like the proposal here would make me more comfortable with relaxing the mixed content restrictions that prohibit unencrypted connections to loopback addresses. Right now, those aren’t really subject to the kinds of concerns that drive most of [MIXED-CONTENT]'s decisions, but I’m reluctant to change Chrome’s implementation without some protection for the local server and user. 

The references to "this would make me more comfortable" and "I’m reluctant to change Chrome’s implementation" sound a bit weird to me for a specification. I'd recommend rewording this so it sounds less personal or implementation-specific decisions.

**Comment by letitz** [comment]
A fair point. This language dates back to the first draft of the spec and is not intended to be final.

**Comment by letitz** [comment]
Reading that paragraph again, I'm not entirely sure what @mikewest was trying to convey.

AIUI there are no "mixed content restrictions that prohibit unencrypted connections to loopback addresses". HTTPS websites can embed stuff from `http://localhost` (which is treated as a secure context) without running into mixed content restrictions.

Mike, do you remember what you had in mind here?

**Comment by mikewest** [comment]
When I wrote that, HTTPS websites could _not_ embed stuff from `http://localhost`. :) I eventually convinced myself and others that the risk here was outweighed by the damage done by developers bundling certs along with their native applications, or installing roots to allow self-signed certs locally (e.g. the historical `https://spotilocal.com`, `https://dropboxlocalhost.com/`, `https://ghconduit.com`), but I didn't update this doc.

So, yes, remove this note. It's not relevant today. :)
